tests/php/Integration/Fixtures/Inline/smaller.test

```diff
@@ -0,0 +1,6 @@
+--PHEL--
+(< 1)
+(< 1 2)
+(< 1 2 3)
+--PHP--
+(\Phel\Lang\Registry::getInstance()->getDefinition("phel\\core", "<"))(1);(1 < 2);(\Phel\Lang\Registry::getInstance()->getDefinition("phel\\core", "<"))(1, 2, 3);
```

Comment:
Although these tests are technically correct, the PHP assertion is really hard to understand because everything is in one line. Could we place them in different lines somehow? I know it might be not that easy, so we (I, for example)  could work on a follow up PR to try this idea out.

Reply:
Yes that can be improved. I just tried it but it is more work. Basically one have to change `OutputEmitter::emitContextSuffix` to do `emitLine` instead of `emitStr` and then remove all unnecessary `emitLine` calls in the other Emitters.